### DISCUSSION TITLE
Cant rename/create/delete/move files from file explorer in WSL2?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [microsoft/WSL#6861] [Q&A]
Im seeing this every time I try creating any new file in `\\wsl$\...`:

![image](https://user-images.githubusercontent.com/20190653/115987659-4b3f4280-a5b6-11eb-870d-84b179116ed2.png)

As well as that files re-appear when I try deleting one.

Im running the Ubuntu marketplace distro.

Any knowledge appreciated <3

**Comment by Biswa96** [comment]
root user owned files are not allowed to delete.
